packages/@react-spectrum/tag/docs/TagGroup.mdx

```diff
@@ -0,0 +1,164 @@
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/tag';
+import {HeaderInfo, PropTable, PageDescription} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/tag/package.json';
+
+```jsx import
+import {Flex} from '@react-spectrum/layout';
+import {Item, TagGroup} from '@react-spectrum/tag';
+import News from '@spectrum-icons/workflow/News';
+import Airplane from '@spectrum-icons/workflow/Airplane';
+import Game from '@spectrum-icons/workflow/Game';
+import ShoppingCart from '@spectrum-icons/workflow/ShoppingCart';
+import {Text} from '@react-spectrum/text';
+```
+
+---
+category: Status
+keywords: [TagGroup, status]
+---
+
+# TagGroup
+
+<PageDescription>{docs.exports.TagGroup.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['Item', 'TagGroup']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/tag/'}
+  ]} />
+
+## Example
+```tsx example
+<TagGroup aria-label="Static TagGroup items example">
+  <Item>Adobe Photoshop</Item>
+  <Item>Adobe InDesign</Item>
+  <Item>Adobe AfterEffects</Item>
+  <Item>Adobe Illustrator</Item>
+  <Item>Adobe Lightroom</Item>
+</TagGroup>
+```
+
+## Content
+TagGroup is a component that allows users to categorize content. Basic usage of TagGroup, seen in the example above, shows the use of a static collection where the contents of the TagGroup are hard coded.
+Dynamic collections, as shown below, can be used when the options come from an external data source such as an API, or update over time.
+
+Each item has a unique key defined by the data. In the example below, the `key` of each row element is implicitly defined by the id property of the row object.
+See [collections](../react-stately/collections.html#unique-keys) to learn more about keys in dynamic collections.
+
+```tsx example
+const items = [
+  {id: 1, name: 'Adobe Photoshop'},
+  {id: 2, name: 'Adobe XD'},
+  {id: 3, name: 'Adobe InDesign'},
+  {id: 4, name: 'Adobe AfterEffects'},
+  {id: 5, name: 'Adobe Illustrator'},
+  {id: 6, name: 'Adobe Lightroom'},
+  {id: 7, name: 'Adobe Premiere Pro'},
+  {id: 8, name: 'Adobe Fresco'},
+  {id: 9, name: 'Adobe Dreamweaver'}
+];
+
+<Flex width={500}>
+  <TagGroup items={items} aria-label="Dynamic TagGroup items example">
+    {(item) => (
+      <Item key={item.key}>
+        {item.name}
+      </Item>
+    )}
```

Comment:
`item.key` doesn't exist. the `id` property will be used automatically, so no `key` prop is needed here.

```suggestion
    {item => <Item>{item.name}</Item>}
```